src/core/client/ui/components/TextField/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./TextField";
```

Comment:
I realized this pattern doesn't always work and isn't future proof (e.g. when testing suddenly requires other exports and we don't to export them here), let's remove this line and explicitly export what we want.